tools/schema/zone.json

```diff
@@ -0,0 +1,31 @@
+{
+    "type": "object",
+    "defaultProperties": [
+        "name",
+        "properties",
+        "location"
+    ],
+    "additionalProperties":false,
+    "properties": {
+        "name": {
+            "required": true,
+            "propertyOrder": 1,
+            "title": "name",
+            "description": "Zone name - so you'll know which zone you're looking at layer.",
+            "type": "string"
+        },
+        "properties": {
+            "propertyOrder": 2,
+            "title": "properties",
+            "description": "A list of properties for this zone, in order. Property groups are identified by the prefix '#'. Properties that are not desired from a group can be removed with the prefix '!', as if it was never there.",
+            "type": "array",
+            "items": {
+                "type": "string"
+            }
+        },
+        "location": {
```

Comment:
Better to change this to "area" now before it's harder to change later :(

Reply:
Oh, well then ignore my comment a file or two up. Changing it :)

Reply:
...actually, second thought, let's do that as a second pull request. Changing code? Maybe. Changing hundreds of json files? Not today if I can help it.

Reply:
Bleh, yeah. Didn't realize I had already goofed this before this pull request.